lookout/style/format/feature_extractor.py

```diff
@@ -289,8 +304,8 @@ def extract_features(self, files: Iterable[File], lines: List[List[int]]=None
                 sibling_indices_list.extend(sibling_indices)
         self._log.debug("Features shape: %s" % (X.shape,))
         if self.return_sibling_indices:
-            return X, y, vnodes_y, vnodes, sibling_indices_list
-        return X, y, vnodes_y, vnodes
+            return X, y, vnodes_y, vnodes, vnodes_parents, parents, sibling_indices_list
+        return X, y, vnodes_y, vnodes, vnodes_parents, parents
```

Comment:
Can you please organize this in this way:

```python
X, y, (vnodes_y, vnodes, vnodes_parents, parents)
```

Also rename `parents` to `node_parents` and `vnodes_parents` to `vnode_parents` + update the docstrings.